### ISSUE TITLE
Accessibility: WCAG2AA.Principle4.Guideline4_1.4_1_2.H91.InputCheckbox.Name

### ISSUE THREAD

**Issue by funkysi1701** [funkysi1701/funkysi1701.github.io#2825]
## Accessibility Issue Found by Pa11y

**WCAG Code**: `WCAG2AA.Principle4.Guideline4_1.4_1_2.H91.InputCheckbox.Name`
**Total Occurrences**: 1
**URL**: https://blog-dev.funkysi1701.com
**Build**: 10.1.1.2333

---

### Example 1

**Message**: This checkboxinput element does not have a name available to an accessibility API. Valid names are: label element, title , aria-label , aria-labelledby .
**Selector**: `#modeSwitcher`

**Context:**
```html
<input class="form-check-input" type="checkbox" id="modeSwitcher">
```

---
_Detected by [Pa11y](https://pa11y.org/) accessibility checker in Azure Pipeline_
